README.md

```diff
@@ -261,24 +261,28 @@ for any `t` such that `t(a).map(f)` is equivalent to `t(a.map(f))` (naturality)
 
 2. `u.map(F.of).sequence(F.of)` is equivalent to `F.of(u)` for any Applicative `F` (identity)
 
-3. `u.map(Compose.of).sequence(Compose.of)` is equivalent to
-   `Compose.of(u.sequence(f.of).map(x => x.sequence(g.of)))` for Compose type defined below (composition)
+3. `u.map(Compose.wrap).sequence(Compose.of)` is equivalent to
+   `Compose.wrap(u.sequence(F.of).map(v => v.sequence(G.of)))` for Compose type defined below (composition)
 
 ```js
 var Compose = function(c) {
   this.c = c;
 };
 
-Compose.of = function(c) {
-  return new Compose(c);
+Compose.wrap = function(c) {
+  return new Compose(c)
+};
+
+Compose.of = function(x) {
+  return new Compose(F.of(G.of(x)));
 };
```

Comment:
I am a little unsettled that this would be a runtime error if someone copy/pasted it. And given that we have issues like #128, we should try to be more aware that what seems obvious to one or two of us, isn't necessarily good for the spec. How do you feel about making these variables explicit?